src/alma-gateway-for-woocommerce.php

```diff
@@ -50,18 +50,111 @@
 	define( 'ALMA_PLUGIN_URL', plugin_dir_url( __FILE__ ) );
 }
 
+// Define a constant to see if site is network activated.
+if ( ! function_exists( 'is_plugin_active_for_network' ) ) {
+	// Makes sure the plugin is defined before trying to use it.
+	require_once ABSPATH . '/wp-admin/includes/plugin.php';
+}
+
+if ( is_plugin_active_for_network( 'alma-gateway-for-woocommerce/alma-gateway-for-woocommerce.php' ) ) {
+	define( 'ALMA_PLUGIN_NETWORK_ACTIVATED', true );
+} else {
+	define( 'ALMA_PLUGIN_NETWORK_ACTIVATED', false );
+}
```

Comment:
You can maybe make define false by default and add the if after ? 
```
define( 'ALMA_PLUGIN_NETWORK_ACTIVATED', false );
if ( is_plugin_active_for_network( 'alma-gateway-for-woocommerce/alma-gateway-for-woocommerce.php' ) ) {
	define( 'ALMA_PLUGIN_NETWORK_ACTIVATED', true );
}
```

Reply:
You can not ovveride a constant with define twice